### DISCUSSION TITLE
Add the ability to unlist repositories

### DISCUSSION THREAD

**Discussion by Ahsoka** [community/community#11758] [Other Feature Feedback, Questions, & Ideas]
The "unlisted" state of a repository would be like the "unlisted" state of a YouTube video.  It would be publicly available but only to those with the link.  So a repository that is unlisted wouldn't be visible on the user's homepage but it would still be viewable via the link without any authentication needed like with a private repository.

**Comment by ben-brook** [comment]
If added, this feature should make the URLs hard to guess and meaningless to a good degree, since as a user I would want my unlisted repositories to only be accessed by those given a URL. The repository should also not be indexable by search engines.

+1 though. I need to share a repository I'm writing as school coursework to my teacher via a URL without sharing the project to the public. It is more convenient and requires less back-and-forth to share a URL to my teacher than get them an account set up and invite them as a collaborator.

**Reply by Lauro235** [reply]
I agree with everything above. Definitely coming to the point where unlisted repositories are a thing.

**Comment by valioiv** [comment]
I'm definitely in for such feature! What are the plans for implementing it?

**Comment by RealHypnoticOcelot** [comment]
+1, I'd love to see this feature

**Comment by 3DCoded** [comment]
+1 would love to see this feature

**Comment by piggjf** [comment]
This feature would be great for sharing school projects with interviewers.  +1

**Comment by 3DCoded** [comment]
For now, a partial workaround would be to use secret gists and not use folders.

**Comment by BenjaminAster** [comment]
+1 This would also be perfect for forks of other repositories where I don't want them to fill up my public "repositories" tab. I could then e.g. in my personal repository readme make a section with "unlisted repositories" so that other people interested could still find them.

**Comment by ghost** [comment]
+1 great feature!
